policy.md

```diff
@@ -226,24 +317,205 @@ Each audit statement MUST be accompanied by documentation of the audit team qual
+When populating the “JSON Array of Partitioned CRLs” with multiple CRL URLs, CA providers must ensure that each CRL contains a critical Issuing Distribution Point extension and the distributionPoint field of the extension must include a UniformResourceIdentifier.
+The value of the UniformResourceIdentifier must exactly match a URL, from which the CRL was accessed, present in the “JSON Array of Partitioned CRLs” field in the CCADB record associated with the Issuing CA Certificate.
+When populating the “Full CRL Issued By This CA” with a single CRL URL, the CRL should not contain an Issuing Distribution Point extension.
+Under normal operating conditions, the CRL URLs provided by CAs in this section must be available such that Apple systems are able to successfully retrieve the current CRL every 4 hours.
+
+--- 
+
+### Mozilla:
+
+When a CA operator fails to comply with any requirement of this policy - whether it be a misissuance, a procedural or operational issue, or any other variety of non-compliance - the event is classified as an incident. CA operators MUST adhere to the CCADB's Incident Reporting Guidelines (IRGs). CA operators MUST update Incident Reports and respond to questions or comments in accordance with the IRGs until the corresponding Bugzilla bug is closed.
+
+Mozilla expects the timely remediation of the problems that caused or gave rise to an incident. In response to incidents, Mozilla MAY further require that the CA operator submit a plan of action with milestones or submit one or more additional audits to provide sufficient assurance that the incident has been remediated. Such audits MAY be expected sooner than the CA operator’s next scheduled audit, and thus MAY be expected to be for a period less than a year.
+
+--- 
+
+[X] Audit reports that are being supplied to maintain a certificate within the Mozilla root store MUST be provided to Mozilla via the CCADB within three months of the point-in-time date or the end date of the period of time.
+
+----
+
+[X] The publicly-available documentation relating to each audit MUST contain the information required by section 5.1 of the CCADB Policy (v.1.3.1) and the CA locations that were or were not audited. Audit reports MUST also contain or be accompanied by the name of the lead auditor and qualifications of the team performing the audit, as required by section 3.2.
+
+If Mozilla determines that an audit provided does not meet the requirements of this policy, then Mozilla MAY require that the CA operator obtain a new audit, at the CA operator's expense, for the period of time in question. Additionally, depending on the nature of concerns with the audit, Mozilla MAY require that the CA operator obtain such an audit from a new auditor.
```

Comment:
Does the CCADB policy revision state that a CA owner might have to obtain a new audit?

Reply:
Addressed in https://github.com/ryancdickson/www.ccadb.org/commit/e3ea8302985642c1294b03289c0bf8d594cb5d2e.